cdc/sink/mq.go

```diff
@@ -106,22 +108,37 @@ func newMqSink(
 			avroEncoder.SetValueSchemaManager(valueSchemaManager)
 			return avroEncoder
 		}
+	case codec.ProtocolCanal:
+		if !config.EnableOldValue {
+			log.Warn("mqSink View all updates as inserts if enable old value is not enabled")
+		}
```

Comment:
Canal adapter can not support upsert statement, so we mush distinguish `update` and `insert`.